.travis.yml

```diff
@@ -2,6 +2,10 @@ language: java
 
 before_install:
   - chmod +x mvnw
+  - mysql -e 'CREATE DATABASE IF NOT EXISTS db_example'
```

Comment:
The Travis stuff was fixed in [#38](https://github.com/SWE-4103-Group-3/Project/pull/38), we can probably just wait until that merges in.

Reply:
We could, I did this because I was expecting to merge first as this bug could potentially block completion of features (unless we made all of our urls have root as parent)